### DISCUSSION TITLE
Add a permission to replant-crops configuration

### DISCUSSION THREAD

**Discussion by GrowingSky63** [PurpurMC/Purpur#1376] [__Dismissed__] [answered]
When this option is on:
`world-settings.default.tools.hoe.replant-crops: true`

I need it to be true, only for a specific hole in LuckPerms. (Vip/Sub Roles)

**Comment by BillyGalbreath** [comment, marked as answer]
For fine grained control you'll want a plugin.

https://modrinth.com/plugin/scythe

**Reply by GrowingSky63** [reply]
I thought I could do this natively, but thanks for the suggestion, I'll take it.
